app/uk/gov/hmrc/hmrcemailrenderer/templates/ats/AtsTemplates.scala

```diff
@@ -36,6 +37,15 @@ object AtsTemplates {
       plainTemplate = txt.annualTaxSummariesMessageAlert.f,
       htmlTemplate = html.annualTaxSummariesMessageAlert.f,
       priority = Some(MessagePriority.Background)
+    ),
+    MessageTemplate.create(
+      templateId = "annual_tax_summaries_message_alert_cy",
```

Comment:
I thought we are going to use `_cy_` suffix as we have already some templates with `_cy` suffixes.